no-kendoui-professional-in-package-manager.md

```diff
@@ -0,0 +1,67 @@
+
+---
+title: Kendo UI Professional Is Not Available in the AppBuilder Package Manager
+description: Kendo UI Professional Is Not Available in the AppBuilder Package Manager When Using a DevCraft License
+type: Troubleshooting
+page_title: Kendo UI Professional Is Not Available in the AppBuilder Package Manager
+slug: no-kendoui-professional-in-package-manager
+position: 0
+tags: devcraft, kendo, package manager, manage packages, appbuilder, license
+ticketid: 1017056, 1017115
+
+---
+
+## Environment 
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package Manager</td>
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>All</td>
+  </tr>
+</table>
+## Description
+When having two subscriptions, the Devcraft subscription gives you the possibility to use the Kendo UI Professional. However, the Kendo UI Professional package cannot be installed in an AppBuilder app using the Package Manager.
```

Comment:
* Use consistent spelling for DevCraft and consider using a variable for it.
* Names of wizards should be enclosed in bold formatting. (Apply as needed, if I haven't mentioned it earlier.)
* to use the Kendo UI Professional. -> to use the Kendo UI Professional package.